src/main/java/com/peerrequest/app/api/CategoriesController.java

```diff
@@ -12,41 +12,45 @@ public class CategoriesController {
 
     @GetMapping(value = "/categories", produces = "application/json")
     public List<Category> getCategories() {
-        return List.of(mockCategory());
+        return List.of(mockUpCategory);
     }
 
     @GetMapping(value = "/categories/{categoryId}", produces = "application/json")
     public Category getCategory(@PathVariable final int categoryId) {
-        return mockCategory();
+        return mockUpCategory;
```

Comment:
When called first: will return null because private method to initialize mock was never called. 
Same may appear for other methods. To fix this issue fast: Set a default constructor and assign the lokal mock with the method mockCategory() in the body of the default constructor.